README.md

```diff
@@ -4,3 +4,6 @@ An example lesson teaching how to bake sourdough cinnamon rolls.
 
 Generated from [_crispy-doodle_](https://github.com/carpentries/crispy-doodle).
 
+## Contact
+
+If you have questions about the project, please open an issue or [contact the author](mailto:example@example.com).
```

Comment:
thank you for the contribution. However it's the wrong email address. it should be example2@example.com

Reply:
```suggestion
If you have questions about the project, please open an issue or [contact the author](mailto:example2@example.com).
```